### ISSUE TITLE
Remove human param from doc and java

### ISSUE THREAD

**Issue by elouanKeryell-Even** [gisaia/ARLAS-server#118]
In gitlab by @sylvaingaudan on Oct 19, 2017, 11:17

Remove human param from doc and java:
```shell
grep -Rl human *
doc/api/API-Explore-definition.md
examples/EXAMPLE.md
examples/requests/aggregateParameters.json
examples/requests/countParameters.json
examples/requests/geoAggregateParameters.json
examples/requests/geoSearchParameters.json
examples/requests/searchParameters.json
src/main/java/io/arlas/server/model/request/Form.java
```

**Comment by elouanKeryell-Even** [comment]
In gitlab by @sfalquier on Oct 20, 2017, 10:29

added ~2424198 label

**Comment by elouanKeryell-Even** [comment]
In gitlab by @sfalquier on Nov 17, 2017, 08:07

changed milestone to %19

**Comment by elouanKeryell-Even** [comment]
In gitlab by @sfalquier on Nov 24, 2017, 08:21

removed milestone

**Comment by elouanKeryell-Even** [comment]
In gitlab by @sfalquier on Dec 8, 2017, 11:55

mentioned in commit e29c71a2da7d83bff7cd70f983b6d4fd723dd603

**Comment by elouanKeryell-Even** [comment]
In gitlab by @sfalquier on Dec 8, 2017, 11:56

mentioned in merge request !121

**Comment by elouanKeryell-Even** [comment]
In gitlab by @sfalquier on Dec 8, 2017, 14:03

added ~2424199 and removed ~2424198 labels

**Comment by elouanKeryell-Even** [comment]
In gitlab by @sfalquier on Dec 8, 2017, 14:03

assigned to @sfalquier

**Comment by elouanKeryell-Even** [comment]
In gitlab by @sfalquier on Dec 15, 2017, 17:48

changed milestone to %23

**Comment by elouanKeryell-Even** [comment]
In gitlab by @sylvaingaudan on Dec 19, 2017, 09:27

closed via merge request !121

**Comment by elouanKeryell-Even** [comment]
In gitlab by @sylvaingaudan on Dec 19, 2017, 09:27

closed via commit 4f95f1330b81a1c6ba89feb7a99ae6b0c08d6a68

**Comment by elouanKeryell-Even** [comment]
In gitlab by @sylvaingaudan on Jan 15, 2018, 16:50

mentioned in commit 4f95f1330b81a1c6ba89feb7a99ae6b0c08d6a68
